src/material-experimental/mdc-slider/slider.scss

```diff
@@ -1,5 +1,6 @@
 @import '@material/slider/mixins';
 @import '../mdc-helpers/mdc-helpers';
+@import '../../cdk/a11y/a11y';
```

Comment:
can you add the a11y lib styles to the deps of the bazel target?

<sup>It currently works due to an implementation limitation in `rules_sass` where transitive deps are allowed)</sup>